client/reducers/all-users.js

```diff
@@ -0,0 +1,23 @@
+import {
+  RECEIVE_ALL_USERS,
+  LOG_OFF,
+  REQUEST_ALL_USERS
+} from '../actions/auth'
```

Comment:
we can probably fit these 5 lines on a single line and still not exceed 80 characters